fbchat/thread_interraction.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: UTF-8 -*-
+
+from __future__ import unicode_literals
+from .listener import Listener
+
+
+class ThreadInterraction(Listener):
+    """Enables the client to interract with and listen on threads"""
+
+    def set_image(self, thread, image):
+        """Set a group's image
+
+        Args:
+            thread (`Thread`): Group whose image to change
+            image (`Image`): New image
+        """
+
+    def on_image_set(self, thread, actor, old_image):
```

Comment:
The `old_x` can be quite useful, but if we haven't cached the thread before, we won't have a way of knowing this? How should this state be signified? Should we just pass ``None``, and then document that this instance might happen?

Reply:
Also notice how I've changed it from the way it was before, so that the passed `thread` object automatically contains the updated information